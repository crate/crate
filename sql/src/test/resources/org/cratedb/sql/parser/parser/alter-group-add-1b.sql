ALTER GROUP ADD TABLE t1 ( c1, c2 )
    TO t2