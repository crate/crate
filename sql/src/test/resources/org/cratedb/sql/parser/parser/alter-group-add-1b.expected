org.cratedb.sql.parser.parser.AlterTableNode@27aed7c1
name: t1
statementType: ALTER TABLE
objectName: t1
compressTable: false
sequential: false
truncateTable: false
purge: false
defragment: false
truncateEndOfTable: false
updateStatistics: false
updateStatisticsAll: false
indexNameForUpdateStatistics: null
tableElementList: 	
	org.cratedb.sql.parser.parser.TableElementList@47d6d441

	[0]:	
	org.cratedb.sql.parser.parser.FKConstraintDefinitionNode@69091c3f
	refTable name : t2
	grouping: true
	constraintName: null
	constraintType: FOREIGN_KEY
	properties: null
	name: null
	elementType: AT_DROP_CONSTRAINT
	columnList: 		
		org.cratedb.sql.parser.parser.ResultColumnList@75a5555a

		[0]:		
		org.cratedb.sql.parser.parser.ResultColumn@3986c40d
		exposedName: c1
		name: c1
		tableName: null
		isDefaultColumn: false
		type: null
		[1]:		
		org.cratedb.sql.parser.parser.ResultColumn@3eb14efc
		exposedName: c2
		name: c2
		tableName: null
		isDefaultColumn: false
		type: null